src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2109,7 +2109,21 @@ protected override void BeginProcessing()
                 //   "Process was not started by this object, so requested information cannot be determined."
                 // Fetching the process handle will trigger the `Process` object to update its internal state by calling `SetProcessHandle`,
                 // the result is discarded as it's not used later in this code.
-                _ = process.Handle;
+                try
+                {
+                    _ = process.Handle;
+                }
+                catch (Win32Exception e)
```

Comment:
Should we check the HRESULT for the specific error rather than treating them all the same?

Reply:
As getting the `Handle` was an attempt to try and set the internal state and not something that is being used in the code I didn't think it was worth setting a specific error code. I can change it but I didn't think it warranted catching a specific error because we aren't doing anything to work around the error.

Reply:
@SteveL-MSFT Here we should rather move in the other direction. Since the only operation we do is getting a handle, we rather need to catch and ignore all exceptions, especially since if there is a question about behavior, we can always display the debug message.
Although I think Win32Exception covers everything that is needed.